src/zarr/core/array.py

```diff
@@ -2380,8 +2385,18 @@ def chunks_initialized(array: Array | AsyncArray) -> tuple[str, ...]:
 def _build_parents(node: AsyncArray | AsyncGroup) -> list[AsyncGroup]:
     from zarr.core.group import AsyncGroup, GroupMetadata
 
-    required_parts = node.store_path.path.split("/")[:-1]
-    parents = []
+    print("path", node.store_path.path)
+
+    if "/" in node.store_path.path:
+        required_parts = node.store_path.path.split("/")[:-1]
+    else:
+        required_parts = []
+    parents = [
+        AsyncGroup(
+            metadata=GroupMetadata(zarr_format=node.metadata.zarr_format),
+            store_path=StorePath(store=node.store_path.store, path=""),
+        )
+    ]
```

Comment:
@TomAugspurger - this was needed to enable `zarr.create(store={}, path="a")` such that the root `zarr.json` is created in addition to `a/zarr.json`.